### DISCUSSION TITLE
Usage of graphql codegen in a JS project?

### DISCUSSION THREAD

**Discussion by wuarmin** [TanStack/query#4230] [Q&A] [answered]
Hey,
is it possible to autogen graphql hooks, using graphql codegen, for a javascript React app, instead of a Typescript React App?

Thanks and best regards

**Comment by wuarmin** [comment]
An answer would be great 😊

**Comment by TkDodo** [comment, marked as answer]
we're not associated with graphql-codegen so I think this is a question you have to ask in their repo?
